Hazel/src/Platform/OpenGL/OpenGLContext.cpp

```diff
@@ -1,9 +1,9 @@
 #include "hzpch.h"
 #include "OpenGLContext.h"
 
-#include <GLFW/glfw3.h>
 #include <glad/glad.h>
-#include <GL/GL.h>
+//#include <GL/GL.h>
```

Comment:
Commented out code.

Reply:
Removed in https://github.com/TheCherno/Hazel/pull/82/commits/833380b615d9c1c2c3d1122b7fda3207a1aaa678.